algorithms/linfa-tsne/README.md

```diff
@@ -0,0 +1,32 @@
+# Naive Bayes
+
+`linfa-tsne` provides a pure Rust implementation of exact and Barnes-Hut t-SNE.
+
+## The Big Picture
+
+`linfa-tsne` is a crate in the [`linfa`](https://crates.io/crates/linfa) ecosystem, an effort to create a toolkit for classical Machine Learning implemented in pure Rust, akin to Python's `scikit-learn`.
+
+## Current state
+
+`linfa-tsne` currently provides an implementation of the following methods: 
+
+- exact solution t-SNE
+- Barnes-Hut t-SNE
+
+It wraps the [bhtsne](https://github.com/frjnn/bhtsne) crate, all kudos to them.
+
+## Examples
+
+There is an usage example in the `examples/` directory. To run, use:
+
+```bash
+$ cargo run --example tsne --features linfa/intel-mkl-system
+```
```

Comment:
I see that some crates include this feature and some not, probably depending on who wrote the command. I don't know whether it is something worth of a note (like: see the list of openblas backends in [link to linfa main page]) or if it is something so small that that we can just leave it like it is

Reply:
the example whiten the data prior to the embedding and requires therefore a LAPACK backend. I added a small note to look at the README front page with a link, but this happens at multiple places.